### PR TITLE
Review TODO

### PR DESCRIPTION
**Pull request by andriy-mudrak**

### CODE REVIEW

File: Components/InputNote.js [hunk 1]
```diff
@@ -11,34 +11,29 @@ export default class InputBottom extends Component
...
                     placeholder="Write a note..."
                     multiline={true}
-                    clearButtonMode="always"
-
                 />
 
-            {/* </View> */}
-            <Button
-          
+            <Button
```
Comment (vtorkalo) [inline Components/InputNote.js:37]: use disabled attribute to prevent pressing when input is empty

File: Components/Note.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        super(props)
+        this.state = {
+            check: false,
+            color: "red"
+        }
+    }
+    checkBoxTest() {
+        if(this.state.check)
+        {
+            this.setState({color:"red"})
```
Comment (vtorkalo) [inline Components/Note.js]: will be more readable write like this:
also don't call setstate twice

let color = this.state.check ? 'red' : 'green';
this.setState({
    color, 
    check: !this.state.check
});

File: App.js [hunk 3]
```diff
@@ -1,38 +1,61 @@
...
-import MainBackground from 'ToDoProject/Components/MainBackground'
-import InputBottom from 'ToDoProject/Components/inputPart'
-import { Header } from 'react-native-elements';
+import TaskList from 'ToDoProject/Components/TaskList'
+import InputNote from 'ToDoProject/Components/InputNote'
 export default class MainPage extends Component {
   constructor(props) {
     super(props);
-    this.state = { text: '' };
+    this.id = 1;
```
Comment (vtorkalo) [inline App.js]: looks like id is not used

File: App.js [hunk 4]
```diff
@@ -1,38 +1,61 @@
...
+      notes: [
+        { key: '1', description: 'task1', isDone: false },
+
+      ]
+    };
   }
 
+  addNote = (text) => {
+    if (text != '') {
+      this.id++;
```
Comment (vtorkalo) [inline App.js]: don't increment id manually
remove this variable.
Instead use index in renderItem method. Example
`renderItem={({ item }) =>
<MyTask index={item.index}    text= {item.text}/>}`

File: App.js [hunk 5]
```diff
@@ -1,38 +1,61 @@
...
+      this.state.notes.push({
```
Comment (vtorkalo) [inline App.js]: don't modify state directly
create new array 
`let notes = [this.state.notes, ... newNote];
this.setState ({notes});`

File: App.js [hunk 6]
```diff
@@ -1,38 +1,61 @@
...
+        key: this.id.toString(),
+        description: text,
+        isDone: false
+      });
+      this.setState({ notes: this.state.notes });
+    }
+  }
+  onDeleteNote = (id) => {
+    for (var i = 0; i < this.state.notes.length; ++i) {
```
Comment (vtorkalo) [inline App.js]: will be easier:
`let notes = this.state.notes.filter(n=>n.key != id);`
`this.setState({notes});`

File: App.js (hunk 4 above)
Comment (andriy-mudrak) [reply App.js]: What can i put in key, if i don't need id?

File: Components/Note.js [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+                        onPress={() => this.props.setDoneTask(this.props.index)}
+                    />
+                    {/* Alert.alert('You need to...') */}
+                    {/* <Text>asdasd{this.props.text}</Text> */}
+                </View>
+
+                <View style={{ flex: 1, flexDirection: "column" }}>
+                    <TouchableOpacity
+
+                        style={{ flex: 1, justifyContent: "center", backgroundColor: this.props.isDone ? 'green' : 'red', borderRadius: 10 }}
```
Comment (vtorkalo) [inline Components/Note.js:29]: for long list of properties will be more readable place each property in new line

Comment (andriy-mudrak) [reply Components/Note.js:29]: I plan to make a stylesheet for everyone in next commit
Everything will be done

**Pull request merged**